File: mobile/src/main/java/com/ataulm/whatsnext/letterboxd/ApiFilmsResponse.java
package com.ataulm.whatsnext.letterboxd;

import com.google.gson.annotations.SerializedName;

import java.util.List;

class ApiFilmsResponse {

    @SerializedName("next")
    String cursorToNextPageOfResults;

    @SerializedName("items")
    List<ApiFilmSummary> filmSummaries;
}

File: letterboxd/src/main/java/com/ataulm/letterboxd/ApiFilmsResponse.kt
package com.ataulm.letterboxd

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
internal class ApiFilmsResponse(
    @field:Json(name = "next")
    val cursorToNextPageOfResults: String,

    @field:Json(name = "items")
    val filmSummaries: List<ApiFilmSummary>
)
